### ISSUE TITLE
Changes to LogEntry may have broken VolumeUtil.switchVolumes

### ISSUE THREAD

**Issue by dlmarion** [apache/accumulo#4004]
TabletManagementIteratorIT is expecting one tablet to be returned when testing the volume replacement code, but two tablets are being returned. One tablet is being returned because there is an error on the TabletServer in the TabletManagementIterator. The error is:

```
2023-12-01T15:28:58,442 [state.TabletManagementIterator] ERROR: Error computing tablet management actions for extent: 4;some split<
java.lang.IllegalArgumentException: java.net.URISyntaxException: Relative path in absolute URI: default:8080/d4441065-72a7-4536-9f4c-d649ecfcec8e
        at org.apache.hadoop.fs.Path.initialize(Path.java:263) ~[hadoop-client-api-3.3.6.jar:?]
        at org.apache.hadoop.fs.Path.<init>(Path.java:221) ~[hadoop-client-api-3.3.6.jar:?]
        at org.apache.accumulo.server.fs.VolumeUtil.switchVolumes(VolumeUtil.java:93) ~[classes/:?]
        at org.apache.accumulo.server.fs.VolumeUtil.volumeReplacementEvaluation(VolumeUtil.java:180) ~[classes/:?]
        at org.apache.accumulo.server.fs.VolumeUtil.needsVolumeReplacement(VolumeUtil.java:143) ~[classes/:?]
        at org.apache.accumulo.server.manager.state.TabletManagementIterator.computeTabletManagementActions(TabletManagementIterator.java:254) ~[classes/:?]
        at org.apache.accumulo.server.manager.state.TabletManagementIterator.consume(TabletManagementIterator.java:210) [classes/:?]
```

Looking at VolumeUtil.switchVolumes, it's trying to create a Hadoop Path object from `default:8080/d4441065-72a7-4536-9f4c-d649ecfcec8e`.

**Comment by dlmarion** [comment]
I'm curious if this is broken in `main` as well. Looking at VolumeUtilTest, there are no `WAL` FileType objects being tested.

**Comment by dlmarion** [comment]
This may be related to the changes in #3939

**Comment by dlmarion** [comment]
Added #4007 as a test in 2.1 for VolumeUtil.switchVolumes. I'm curious if this test fails on the merge to `main`.

**Comment by dlmarion** [comment]
Merged #4007 from 2.1 -> elasticity with no real issues. I *think* what happened is that even though #3939 was committed before the changes to the TabletManagementIteratorIT code, it was merged to `elasticity` **after** #3893. #3893 introduced some code that tested the volume replacements, and used a relative path with host and port. This test succeeded at the time, but began failing with #3939 was merged from `main` to `elasticity`. 

I created #4010 to fix the test and resolve this issue.

I still think that the comment at https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/tabletserver/log/LogEntry.java#L45 is wrong.
